rel/ci-sys.config

```diff
@@ -1,7 +1,7 @@
 [
  {lager, [
           {handlers, [
-                      {lager_console_backend, [debug,
+                      {lager_console_backend, [info,
```

Comment:
If we're doing this there should also be a debug backend going to file and have debug.log + error.log uploaded as CI artefacts so that they can be downloaded fully.